erizo/src/erizo/dtls/DtlsSocket.cpp

```diff
@@ -250,6 +250,28 @@ void DtlsSocket::computeFingerprint(X509 *cert, char *fingerprint) {
   }
 }
 
+void DtlsSocket::handleTimeout() {
+  (void) BIO_reset(mInBio);
+  (void) BIO_reset(mOutBio);
+  if (DTLSv1_handle_timeout(mSsl) > 0) {
```

Comment:
Yes it will not trigger a retransmission until SSL timeout kicks in. But we check every 1 second (this duration could be shortened) so at the next check SSL will resend the packet. 
The very very few examples about DTLS with openSSL do something similar, they call DTLSv1_handle_timeout in every iteration of the polling loop or the event loop.

Reply:
I'll write down the `DTLSv1_get_timeout` as a potential optimization but I do see how this will work 👍